test/e2e/framework/providers/azure/azure.go

```diff
@@ -63,28 +63,6 @@ func (p *Provider) DeleteNode(node *v1.Node) error {
 	return errors.New("not implemented yet")
 }
 
-// CreatePD creates a persistent volume
-func (p *Provider) CreatePD(zone string) (string, error) {
```

Comment:
Please keep it working, as long as we have Azure cloud provider in the tests. Let's remove it with the cloud provider.

Reply:
fixed